broker/session.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, **kwargs):
         host = kwargs.get("hostname", "localhost")
         user = kwargs.get("username", "root")
         sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        if timeout := kwargs.get("timeout"):
+            sock.settimeout(timeout)
```

Comment:
We will always have a timeout value since we are setting the default from `settings.yaml` here and unless someone explicitly set it to None from the `Host` object or `connect{)` def.

So simply we can:
```suggestion
        sock.settimeout(kwargs.get("timeout"))
```

Reply:
Your suggestion works in general since `settimeout` can accept a value of `None`